seed/views/00602_ldu_court_reporter_overview.js

```diff
@@ -0,0 +1,24 @@
+exports.seed = function (knex, promise) {
+  var sql = `CREATE VIEW app.ldu_court_reporter_overview
+    WITH SCHEMABINDING
+    AS
+    SELECT
```

Comment:
This file and region outside the scope of this story?

Reply:
Needed a way to navigate down to the team level, easiest way was to add the views for national and regional and ldu (they are very similar to the team view)